samples/nrf9160/coap_client/README.rst

```diff
@@ -3,12 +3,24 @@
 nRF9160: nRF CoAP Client
 ########################
 
-The nRF CoAP Client sample demonstrates how to receive data from a public CoAP server with an nRF9160 SiP.
+The nRF CoAP Client sample demonstrates the communication between a public CoAP server and an nRF9160 SiP which acts as the CoAP client.
 
 Overview
-*********
+********
 
-The sample connects to a public CoAP test server, sends periodic GET request for a test resource that is available on the server, and prints the data that is received.
+The nRF CoAP Client sample performs the following actions:
+
+#. Connect to the configured public CoAP test server (indicated by the parameter ``CONFIG_COAP_SERVER_HOSTNAME``)
+#. Send periodic GET request for a test resource (indicated by the parameter ``CONFIG_COAP_RESOURCE``) that is available on the server
+#. Display the received data about the resource on a terminal emulator.
```

Comment:
This bullet point has a period at the end, while the others don't. It should be consistent. (I would go for periods, but it's a bit of a gray zone.)